### PR TITLE
dplan 4020 refactor controller

### PR DESCRIPTION
**Pull request by sierrazamudiodemos**
### Ticket
https://demoseurope.youtrack.cloud/issue/DPLAN-4020/3-5-Als-Bearbeiterin-mochte-ich-eigene-Karten-zur-Planung-einbinden-konnen-um-Lagebeziehungen-oder-nicht-unmittelbar-erkennbare

Description
1. I created a new action in the controller: DemosPlanDocumentController::saveImportedElementsAdminAction
- This action is called in the template elements_admin_import.html.twig
- More exactly, it is called when the user clicks "save" button
- What this action does is to save the imported documents
2. That action then redirects to the existing one: DemosPlanDocumentController::elementAdminListAction
- I adjusted the existing one inspired on the work that you did on DemosPlanMapController

### How to review/test
- Case 1: Go to diplanfest  and bobsh --> verfahren/PROCEDURE_ID/verwalten/planunterlagen
- import document
- cancel import and play around with it.
- It should work as expected

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [ ] Link all relevant tickets
- [ ] Move the tickets on the board accordingly
- [ ] Data-Cy attributes added/updated ([conventions](https://dplan-documentation.demos-europe.eu/development/guidelines-conventions/coding-styleguides/twig_html.html#guideline-for-naming-cypress-hooks))

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/Controller/Document/DemosPlanDocumentController.php
Comment (salisdemos) [inline demosplan/DemosPlanCoreBundle/Controller/Document/DemosPlanDocumentController.php:679]: Am I right, that the `'/verfahren/{procedure}/verwalten/planunterlagen` elementAdminListAction-Controller now would look more or less the same as the `/verfahren/{procedure}/verwalten/karte` elementAdminListAction-Controller and one of them can be deleted in the End?

**Review by salisdemos — COMMENTED** [review]
First of all. For me it looks really good.
But I like to have a second pair of eyes on that from a backend person.
And: I think this is more an improvement than part of the US. So we should not merge this before Featurefreeze and the point it to main. Or is there anything within here, that is required for the Feature?

File: demosplan/DemosPlanCoreBundle/Controller/Document/DemosPlanDocumentController.php
Comment (sierrazamudiodemos) [reply demosplan/DemosPlanCoreBundle/Controller/Document/DemosPlanDocumentController.php:679]: hey @salisdemos exactly, one of them can be deleted, I would recommend to delete the /verwalten/karten

File: config/menus.yml
Comment (martendemos) [inline config/menus.yml:329]: I guess the parameter contains a string? (procedureId)
Therefore I would prefer the name "procedureId" if possible.

Comment (sierrazamudiodemos) [reply config/menus.yml:329]: hey, this url is used in another templates that are not touched in this PR, if I adjust it, I will have to adjust the other templates (which is more a FE thing) I can do it but then it becomes a bit of a bigger scope

**Review by salisdemos — APPROVED** [review]

**Review by salisdemos — APPROVED** [review]

**Review by martendemos — DISMISSED** [review]
got an permission denied when I try to access the "/verwalten/planunterlagen" route.

**Review by salisdemos — APPROVED** [review]

**Comment by salisdemos** [comment]
we doublechecked the concerns from @martendemos so it should be fine to overrule him.

**Pull request merged**
